voice/methods/calls/getCallsCallId.md

```diff
@@ -74,7 +74,19 @@ curl -X GET \
 {% sample lang="csharp" %}
 
 ```csharp
-// coming soon
+var response = voiceClient.GetCallState(accountId, callID);
+Console.WriteLine(response.State);
+```
+
+{% sample lang="java" %}
+
+```java
+ApiResponse<ApiCallStateResponse> response = null;
+try {
+    response = voiceController.getCallState(accountId, callId);
+} catch (IOException | ApiException e) {
+    System.out.println(e.getMessage());
+}
```

Comment:
Idea that just popped in my head. Instead of having all of our docs have these try catch blocks, maybe we should have an error handling section in the sdk page. Like "Here's how to catch a 4XX from the messaging API, here's how to catch a 4XX from the voice API" and so on. Just a thought